### PR TITLE
Issue 68, 74: GNAT CE 2020 support

### PR DESCRIPTION
**Pull request by jklmnn**
- Support for GNAT CE 2020
 - Support for Big_Integers

@treiher please check that this version still works with RecordFlux.

### CODE REVIEW

File: src/minimal/a-nbnbin.adb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (treiher) [inline src/minimal/a-nbnbin.adb]: Is there a reason for having an empty first line in these files?

**Review by treiher — COMMENTED** [review]
da1554d2f2dd273936c267cddf0624a78d59f7b8 works fine with RecordFlux on my local system.

File: src/minimal/a-nbnbin.ads [hunk 2]
```diff
@@ -0,0 +1,192 @@
+
```
Comment (senier) [inline src/minimal/a-nbnbin.ads:9]: We should have a file header in those files.

**Review by senier — CHANGES_REQUESTED** [review]

File: src/minimal/a-nbnbin.ads (hunk 2 above)
Comment (jklmnn) [reply src/minimal/a-nbnbin.ads:9]: Fixed by https://github.com/Componolit/ada-runtime/pull/77/commits/1694bd3dfa66c1ad7748a6c6fe644599844427b6.

File: src/minimal/a-nbnbin.adb (hunk 1 above)
Comment (jklmnn) [reply src/minimal/a-nbnbin.adb]: Fixed by https://github.com/Componolit/ada-runtime/pull/77/commits/1694bd3dfa66c1ad7748a6c6fe644599844427b6.

**Review by treiher — APPROVED** [review]

**Review by senier — APPROVED** [review]

**Pull request merged**